src/main/kotlin/com/terraformation/backend/accelerator/db/VoteStore.kt

```diff
@@ -34,13 +35,27 @@ class VoteStore(
     }
   }
 
-  fun delete(projectId: ProjectId, phase: CohortPhase, userId: UserId? = null) {
+  fun fetchAllVoteDecisions(projectId: ProjectId): List<VoteDecisionModel> {
+    requirePermissions { readProjectVotes(projectId) }
+    return with(PROJECT_VOTE_DECISIONS) {
+      dslContext
+          .select(PROJECT_VOTE_DECISIONS.asterisk())
+          .from(PROJECT_VOTE_DECISIONS)
+          .where(PROJECT_ID.eq(projectId))
+          .orderBy(PROJECT_ID, PHASE_ID)
+          .fetch { VoteDecisionModel.of(it) }
+    }
+  }
+
+  fun delete(projectId: ProjectId, phase: CohortPhase, userId: UserId? = null): VoteDecisionModel {
```

Comment:
Might be worth a documentation comment clarifying that this deletes _voters_ from the project+phase, as opposed to deleting (setting to null) their _votes_.